packages/editor/src/core/Provider/index.tsx

```diff
@@ -137,23 +144,32 @@ const Provider: React.FC<ProviderProps> = ({
 
   // prevent options from recreating all the time
 
+  const uiTranslatorMemorized = useMemo(
+    () => ({
+      t: uiTranslator,
```

Comment:
this function here should handle the case, where `uiTranslator` returns nothing. It should return a falllback value.

Reply:
As I mentioned in the previews comment just above, depends on what you want, `uiTranslator` could just return key or `undefined`.